lesson25/src/js/validation.js

```diff
@@ -0,0 +1,67 @@
+const showErrorMessage = (target,message) => {
+    target.classList.add("border-2","border-red-500");
+
+    const errorMessage = document.createElement("p");
+    errorMessage.classList.add("field-invalid","text-sm","my-2","text-rose-600");
+	errorMessage.textContent = message;
+
+	const error = target.nextElementSibling;
+	if(!error){
+		target.insertAdjacentElement("afterend",errorMessage);
+	}else{
+		if(!(error.classList.contains("field-invalid"))){
+			target.insertAdjacentElement("afterend",errorMessage);
+		}
+	}
```

Comment:
Q.
Is this not good enough?
This is working on my end.
If you can tell me if there's any inconvenience, I'd love to learn!
```suggestion
	if(!error){
		target.insertAdjacentElement("afterend",errorMessage);
	}
```